### ISSUE TITLE
Feature Request: add "weekly downloads" column to npm package browser

### ISSUE THREAD

**Issue by katerlouis** [bdkjones/CodeKit#561]
**Quick, short summary:**
I miss the "star count" from good old bower times;
it helps a lot in verifying you use the correct package

also sorting by weekly downloads, just like with the stars, would be nice

**Expected results:**

**Actual results:**

**Exact steps to reproduce:**

**A link to download a simplified project or file that shows the issue:**

**Your configuration (any details about your system that you think might be relevant)**

**Comment by bdkjones** [comment]
I agree; that would be very useful. Unfortunately, npm's API is....what it is. 

I *can* get download-count information, but doing so would require a separate API request for *each* package. That information is not part of the "summary" information npm returns about a package:

https://github.com/npm/registry/blob/master/docs/REGISTRY-API.md

There *is* an API for download counts (https://github.com/npm/registry/blob/master/docs/download-counts.md), but the trouble is that npm also rate-limits its APIs and if I fire off hundreds of requests to get download counts, you'd quickly find that npm refuses to handle any more requests from CodeKit for a while.

So, the answer here is: I'm waiting on npm to improve. 

What you'll want to do instead is find and verify a package ONCE, then add it as a favorite so that you don't have to do that work again.
